user_modules/shared_testcases/testCasesForExternalUCS.lua

```diff
@@ -85,29 +88,29 @@ local utils = { }
     utils.pts.policy_table.module_config.preloaded_pt = nil
   end
 
-	local function ptu(test, id, status)
+  local function ptu(test, status)
     local policy_file_name = "PolicyTableUpdate"
     local policy_file_path = commonFunctions:read_parameter_from_smart_device_link_ini("SystemFilesPath")
     local ptu_file_name = os.tmpname()
     local requestId = test.hmiConnection:SendRequest("SDL.GetURLS", { service = 7 })
     EXPECT_HMIRESPONSE(requestId)
     :Do(function()
-      test.hmiConnection:SendNotification("BasicCommunication.OnSystemRequest",
-        { requestType = "PROPRIETARY", fileName = policy_file_name })
-      utils.createJsonFileFromTable(utils.pts, ptu_file_name)
-      EXPECT_HMINOTIFICATION("SDL.OnStatusUpdate", { status = "UPDATING" }, { status = status }):Times(2)
-      test["mobileSession"..id]:ExpectNotification("OnSystemRequest", { requestType = "PROPRIETARY" })
-      :Do(function()
-	      local corIdSystemRequest = test["mobileSession"..id]:SendRPC("SystemRequest",
-	        { requestType = "PROPRIETARY", fileName = policy_file_name }, ptu_file_name)
-	      EXPECT_HMICALL("BasicCommunication.SystemRequest")
-	      :Do(function(_, d)
-	        test.hmiConnection:SendResponse(d.id, "BasicCommunication.SystemRequest", "SUCCESS", { })
-	        test.hmiConnection:SendNotification("SDL.OnReceivedPolicyUpdate",
-	          { policyfile = policy_file_path .. "/" .. policy_file_name })
-	        end)
-	        test["mobileSession"..id]:ExpectResponse(corIdSystemRequest, { success = true, resultCode = "SUCCESS"})
-        end)
+        test.hmiConnection:SendNotification("BasicCommunication.OnSystemRequest",
+          { requestType = "PROPRIETARY", fileName = policy_file_name })
+        utils.createJsonFileFromTable(utils.pts, ptu_file_name)
+        EXPECT_HMINOTIFICATION("SDL.OnStatusUpdate", { status = "UPDATING" }, { status = status }):Times(2)
+        test.mobileSession1:ExpectNotification("OnSystemRequest", { requestType = "PROPRIETARY" })
```

Comment:
@istoimenova, I've got the point.
PT Updating is supposed only for 1st single application